common-ui/src/main/res/values-night/widget-colors.xml

```diff
@@ -33,10 +33,12 @@
 
 <resources>
     <!-- Widget colors -->
-    <color name="favoriteTitleTextColor">@color/white</color>
-    <color name="favoriteFaviconBackgroundColor">@color/almostBlackDark</color>
-    <color name="searchWidgetBackgroundColor">@color/newBlack</color>
-    <color name="searchWidgetSearchBarBackgroundColor">@color/almostBlack</color>
-    <color name="searchWidgetSearchBarHintColor">@color/white</color>
-    <color name="searchWidgetSearchBarLoupeColor">@color/white</color>
+    <color name="favoriteTitleTextColor">@color/white60</color>
```

Comment:
this is great, can we do this for all the different @color used in the resources?

Reply:
I removed the ones we are no using anymore after removing them from widgets. 
I suppose we can remove all resources in a different task. I wouldn't like to maybe add some bugs here after design review is done for this task.